Documentation/ceph-fs-subvolumegroup.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: SubVolumeGroup CRD
+title: Subvolume Group CRD
```

Comment:
This tracks better with the naming of the CRD being `SubVolume`.

```suggestion
title: SubVolume Group CRD
```

Reply:
It seemed more readable that way, but it's subtle enough probably better to match the CRD name.